tsconfig.json

```diff
@@ -6,6 +6,7 @@
     "moduleResolution": "node",
     "noEmit": true,
     "skipLibCheck": true,
-    "resolveJsonModule": true
+    "resolveJsonModule": true,
+    "strict": true
```

Comment:
you will probably have to fix other files too, but let's do it one by one 😄 nice work!